packages/keyboard-key/src/index.ts

```diff
@@ -303,54 +303,57 @@ const keyboardKeyDefinition = {
    * Mapping from numeric key code to key name. If the value is an array, the first element is the
    * primary key name, and the second element is the key name when shift is pressed.
    */
-  codes: codes,
+  codes: codes
```

Comment:
Do  we actually need an object with a `codes` property or can `codes` be exported as a separate const?

Reply:
Funny for you to write this. It is specifically what I'm trying right now to see how much bundle size I can shave off 😆.

Reply:
Wait why? 😆

Reply:
I meant exporting `codes` as a separate const.

Reply:
Ahhh I incorrectly parsed your comment yesterday :)